src/tfr_executive/src/clock_service.cpp

```diff
@@ -61,7 +61,9 @@ class ClockService
         bool timeRemaining(tfr_msgs::DurationSrv::Request &req,
                 tfr_msgs::DurationSrv::Response &res)
         {
-            res.duration = mission_duration - (ros::Time::now() - mission_start);
+            if (!mission_start.isValid())
+                ROS_WARN("Clock Service: Uninitialized Mission Clock Detected");
+             res.duration = mission_duration - (ros::Time::now() - mission_start);
```

Comment:
There appears to be an extra space on this line, which is why Git is detecting a diff.

Reply:
copy pasta error